docs/api/en/core/Object3D.html

```diff
@@ -53,8 +53,8 @@ <h3>[property:Material customDistanceMaterial]</h3>
 
 		<h3>[property:Boolean frustumCulled]</h3>
 		<p>
-		When this is set, it checks every frame if the object is in the frustum of the camera before rendering the object.
-		Otherwise the object gets rendered every frame even if it isn't visible. Default is *true*.
+		When this is set, the object is only rendered when inside the camera frustum.
+		Otherwise the object gets rendered every frame unless it isn't visible. Default is *true*.
```

Comment:
I think part of the ambiguity here comes from interpreting "isn't visible" as meaning "visible to the camera" vs. "`.visible = true`. Would "unless `object.visible` is false" work?

Reply:
Yes I always read "visible" here as meaning "visible to the camera". I think the line can be changed to something like the following:

> Otherwise the object gets rendered even if it isn't in view.

Ideally the docs for `frustumCulled` wouldn't have to call out every field that impacts whether or not and object is rendered.

Reply:
"When this is set, the object is only rendered when inside the camera frustum (Default is *true*).
Otherwise the object gets rendered every frame unless object.visible is false."

Sounds the cleanest to me.

Reply:
> (Default is true)

This wording is normally always placed at the end of the description.

Reply:
Ok did not know about that. Then how about: "When this is set, the object is only rendered when it is visible and inside the camera frustum. Otherwise the object gets rendered every frame. (Default is true)"

Reply:
@SeanPedersen Do you mind updating the PR with your suggested description?

Reply:
It seems I can't do it using Github.com. I don't mind if this pull requests gets deleted and the new description finds it's way into the doc another way. I could create a new PR if needed via github.com